docs/guides/implementing-connectors.md

```diff
@@ -0,0 +1,88 @@
+---
+id: implementing-connectors
+title: Implement connectors
+description: "Connectors communicate with any system or technology, reducing the time it takes to automate and orchestrate business processes across systems."
+keywords: [connector, modeling, connectors]
+---
+
+The launch of [Camunda Platform 8](../components/concepts/what-is-camunda-platform-8.md) also introduced an integration framework with a key goal: integrate faster to reduce the time it takes to automate and orchestrate business processes across systems.
+
+This goal is achieved using [connectors](../components/modeler/web-modeler/connectors/index.md). Ready to use out of the box, connectors help automate complex [business processes](../components/concepts/processes.md) by inserting them into [BPMN diagrams](./automating-a-process-using-bpmn.md) within [Web Modeler](../components/modeler/about.md), and configuring them via the properties panel.
+
+Connectors technically consist of two parts: the business logic is implemented as a job worker, and the user interface during modeling is provided using an element template. In this guide, we'll walk step-by-step through the implementation of a sample connector.
```

Comment:
Link to some content on the job worker, either the glossary or concept, since it keeps coming up in beginner-to-C8 conversations.

https://docs.camunda.io/docs/reference/glossary/#job-worker
https://docs.camunda.io/docs/components/concepts/job-workers/